src/app/features/Chanels/service/index.js

```diff
@@ -0,0 +1,14 @@
+import { channelsMockData, postsMockData } from '../../../utils/mockData';
+
+const requestChannels = async () => {
+  // eslint-disable-next-line no-return-await
+  return await channelsMockData;
```

Comment:
it is still not Async :D. Please, wrap this in Promise and use setTimeout of 300ms to simulate async behavior

Reply:
But, it's not mandatory. You can keep it like this, though

Reply:
Just, if you want to really make sure, that async actions work